model/task_queue_service.go

```diff
@@ -324,6 +324,7 @@ func (t *basicCachedDispatcherImpl) FindNextTask(spec TaskSpec) *TaskQueueItem {
 				"schedulable_unit_running_hosts":        unit.runningHosts,
 				"schedulable_unit_max_hosts":            unit.maxHosts,
 				"num_schedulable_unit_task_queue_items": len(unit.tasks),
+				"task_id_returned":                      &unit.tasks[0].Id,
```

Comment:
I'm worried that this could panic if len(unit.tasks) == 0, also there's no need to take the pointer to it, I think.